packages/twenty-front/src/modules/command-menu/pages/page-layout/utils/filterSortOptionsByFieldType.ts

```diff
@@ -0,0 +1,51 @@
+import { type IconComponent } from 'twenty-ui/display';
+import { GraphOrderBy } from '~/generated/graphql';
+
+export type SortOption = {
+  value: GraphOrderBy;
+  icon?: IconComponent | null;
+};
+
+type FilterSortOptionsParams = {
+  options: SortOption[];
+  isSelectField: boolean;
+  isDateField?: boolean;
+  chartType: 'bar' | 'line' | 'pie';
+};
+
+export const filterSortOptionsByFieldType = ({
+  options,
+  isSelectField,
```

Comment:
Can't we create a string literal type instead ? Seems like we can put isSelectField and isDateField to true which doesn't make sense ?

Reply:
Drilling down a field type seems appropriate